site/content/docs/5.3/migration.md

```diff
@@ -158,6 +158,9 @@ Please use Boosted v5.3.2 to have the dark mode available.
 - **Offcanvas**
   - <span class="badge bg-success">New</span> A tooltip was added on close buttons of offcanvases. Please add this tooltip in your websites, by adding the attributes `data-bs-toggle`, `data-bs-placement` and `data-bs-title`.
 
+- **Orange Navbar**
+  - <span class="badge bg-warning">Warning</span> The minimizing behavior with `.header-minimized` is applied on top of the `<header>` instead of one of his child directly.
```

Comment:
```suggestion
  - <span class="badge bg-warning">Warning</span> The minimizing behavior with `.header-minimized` is applied to the `<header>` instead of one of its children directly.
```